src/psyclone/psyGen.py

```diff
@@ -2717,6 +2717,8 @@ def module_inline(self, value):
         my_schedule = self.ancestor(InvokeSchedule)
         for kernel in my_schedule.walk(Kern):
```

Comment:
Not your fault but pycodestyle reports that lines 1268-1271 are too long. Would you mind fixing?

Reply:
Fixed